.github/workflows/ci-main.yml

```diff
@@ -19,37 +19,30 @@ jobs:
     steps:
       - name: Freeing up more disk space
         run: |
-          free -h
           sudo rm -rf /usr/local/lib/android # will release about 10 GB if you don't need Android
           sudo rm -rf /usr/share/dotnet # will release about 20GB if you don't need .NET
           sudo rm -rf /opt/ghc
           sudo rm -rf /usr/local/share/boost
           sudo rm -rf /opt/hostedtoolcache/CodeQL
-          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
-          sudo apt-get remove -y 'php.*' --fix-missing
-          sudo apt-get remove -y '^mongodb-.*' --fix-missing
-          sudo apt-get remove -y '^mysql-.*' --fix-missing
-          sudo apt-get clean          
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"     
           df -h
       - uses: actions/checkout@v2
 
-      - name: Install package
+      - name: Install sccache (macos-latest)
         run: |
-          sudo docker image prune --all --force
-          echo 'APT::Get::Always-Include-Phased-Updates "false";' | sudo tee /etc/apt/apt.conf.d/99-phased-updates
-          sudo apt-get update && sudo apt-get upgrade -y
-          sudo apt-get install -y protobuf-compiler libprotobuf-dev
+          brew update
+          brew install sccache     
 
       - name: Setup stable Rust toolchain
         if: matrix.rust == 'stable'
         # Call `rustup show` as a hack so that the toolchain defined in rust-toolchain.toml is installed
         run: rustup show
 
-      - name: Remove rust-toolchain.toml for nightly
-        if: matrix.rust == 'nightly'
-        # To make sure that the nightly version will be used all throughout
-        run: |
-          rm /home/runner/work/spacewalk/spacewalk/rust-toolchain.toml
+      # - name: Remove rust-toolchain.toml for nightly
+      #   if: matrix.rust == 'nightly'
+      #   # To make sure that the nightly version will be used all throughout
+      #   run: |
+      #     rm /home/runner/work/spacewalk/spacewalk/rust-toolchain.toml
```

Comment:
Are we sure that this is not needed anymore? Or did you remove it because the path in l. 45 was not found?

Reply:
No, it was required. was commented out just to test but has been re-added.